CollAction/Services/Newsletter/NewsletterSubscriptionService.cs

```diff
@@ -59,6 +60,7 @@ public NewsletterSubscriptionService(IOptions<NewsletterSubscriptionServiceOptio
             _newsletterListId = options.Value.MailChimpNewsletterListId;
             _jobClient = jobClient;
             _logger = logger;
+            _userId = options.Value.MailChimpUserId;
```

Comment:
If it's not used in this service, it should be added as a variable.

Reply:
Do you mean as a variable in Index.cshtml?

Reply:
No, I mean _userId. It looks like a new variable, and it's not used anywhere, so it's redundant.

Reply:
I see. I removed it